python/rmm/rmm/python/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019-2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .device_buffer import DeviceBuffer
```

Comment:
Why is this here? (I'm a bit of a noob on __init__).

Reply:
It lets you access `DeviceBuffer` from the `python` subpackage instead of having to go through the `device_buffer` module. So `from rmm.python import DeviceBuffer` vs `from rmm.python.device_buffer import DeviceBuffer`